### DISCUSSION TITLE
ec2 IAM role  (via iam profile)

### DISCUSSION THREAD

**Discussion by dzirg44** [vmware-tanzu/velero#3541] [Community support Q&A]
Hi!
As I understood this functionality isn't implemented. I mean , when I bound specific permissions for ec2 instance via iam profile\roles. Am I right? 
 Do we have related topics\discussions about it? I tried to find it without any success. (( 
Thank you in  advance for your help.

**Comment by jenting** [comment]
Not quite understand your question, but are you asking for this https://github.com/vmware-tanzu/velero-plugin-for-aws#option-2-set-permissions-using-kube2iam?
